features/ec2/step_definitions/ec2.js

```diff
@@ -1,175 +0,0 @@
-  });
-  callback();
-});
-
-Given("I describe the EC2 instance {string}", function (instanceId, callback) {
-  this.request(null, "describeInstances", { InstanceIds: [instanceId] }, callback, false);
-});
-
-Given(
-  "I attempt to copy an encrypted snapshot across regions",
-  copySnapshotIntegration({ encryptSource: true, encryptDestination: false })
-);
-
-Given(
-  "I attempt to copy and encrypt an unencrypted snapshot across regions",
-  copySnapshotIntegration({ encryptSource: false, encryptDestination: true })
-);
-
-Given(
-  "I attempt to copy and encrypt an encrypted snapshot across regions",
-  copySnapshotIntegration({ encryptSource: true, encryptDestination: true })
-);
-
-Then("the copy snapshot attempt should be successful", function (callback) {
-  this.assert.equal(this.success, true);
-  callback();
-});
-
-function copySnapshotIntegration({ encryptSource, encryptDestination } = {}) {
-  return async function () {
-    let volId, srcSnapId, dstSnapId, sourceKmsKey, destinationKmsKey;
-    const sourceRegion = "us-west-2";
-    const destRegion = "us-east-1";
-    const srcEc2 = new EC2({ region: sourceRegion });
-    const dstEc2 = new EC2({ region: destRegion });
-
-    const sourceKeyAlias = "js-sdk-e2e-test-key-1-source";
-    const destinationKeyAlias = "js-sdk-e2e-test-key-2-destination";
-
-    const sourceKms = new KMS({ region: sourceRegion });
-    const destinationKms = new KMS({ region: destRegion });
-
-    function teardown() {
-      if (volId) srcEc2.deleteVolume({ VolumeId: volId });
-      if (srcSnapId) srcEc2.deleteSnapshot({ SnapshotId: srcSnapId });
-      if (dstSnapId) dstEc2.deleteSnapshot({ SnapshotId: dstSnapId });
-    }
-
-    try {
-      if (encryptSource) {
-        sourceKmsKey = await getKey(sourceKms, sourceKeyAlias);
-      }
-      if (encryptDestination) {
-        destinationKmsKey = await getKey(destinationKms, destinationKeyAlias);
-      }
-
-      const createVolume = await srcEc2.createVolume({
-        AvailabilityZone: sourceRegion + "a",
-        TagSpecifications: [
-          {
-            Tags: [{ Key: "aws-sdk-js-integration", Value: "SAFE_TO_DELETE" }],
-            ResourceType: "volume",
-          },
-        ],
-        Size: 1,
-        Encrypted: encryptSource ? true : undefined,
-        KmsKeyId: encryptSource ? sourceKmsKey : undefined,
-      });
-
-      volId = createVolume.VolumeId;
-
-      await waitUntilVolumeAvailable({ client: srcEc2, maxWaitTime: 120 }, { VolumeIds: [volId] });
-      const createSnapshot = await srcEc2.createSnapshot({
-        VolumeId: volId,
-        TagSpecifications: [
-          {
-            Tags: [{ Key: "aws-sdk-js-integration", Value: "SAFE_TO_DELETE" }],
-            ResourceType: "snapshot",
-          },
-        ],
-      });
-
-      srcSnapId = createSnapshot.SnapshotId;
-
-      await waitUntilSnapshotCompleted(
-        { client: srcEc2, maxWaitTime: 300 },
-        {
-          SnapshotId: createSnapshot.SnapshotId,
```

Comment:
this waiter also had the same typo, but for whatever reason passed. Must be a false positive, with `SnapshotIds[]` considered empty and passing.